app/views/stats_graph.php

```diff
@@ -0,0 +1,41 @@
+<div id="graphdiv"></div>
+
+<script src="https://l10n.mozilla-community.org/~pascalc/web_l10n_stats/dygraph-combined.js"></script>
+
+<script>
+graph = new Dygraph(
+    document.getElementById("graphdiv"),
+    "https://l10n.mozilla-community.org/~pascalc/web_l10n_stats/logs/<?=$locale?>.csv", // path to CSV file
+    {
+        gridLineColor: 'lightgray',
+        highlightCircleSize: 5,
+        strokeWidth: 1,
+        title: 'Translation progress of lang files over time',
+        valueRange: [0, 1900],
```

Comment:
Drop this line and let the axis auto-size. On active locales the graph is really small with the axis set so high.